chaindexing-postgres/src/repo.ts

```diff
@@ -55,6 +55,23 @@ export class PostgresRepo extends Repo<Pool, Conn> {
     // TODO: Implement serial streaming version
   }
 
+  async streamContractAddresses_2(conn: Conn, limit = 10, allowOffset = true) {
```

Comment:
To keep it consistent with the Rust's lib:

```suggestion
  async getContractAddressesStream(conn: Conn) {
```

Reply:
We can hardcode the internal parameters (`limit`, `allowOffset`,...)